core/src/main/resources/config.yml

```diff
@@ -152,6 +152,17 @@ forwardSkin: true
 # If they still want to invoke the command, they have to invoke /premium again
 premium-warning: true
 
+# When set to true, enables the use of alternative session resolver which does not send the server IP
+# to mojang session servers. This setting might be useful when you are trying to run the server via a
+# transparent reverse proxy or some other form of DNAT. As far as security goes, this setting has
+# negligible to no security impact.
```

Comment:
Might be necessary to mention this is only related to Spigot protocollib feature and corresponds to the `prevent-proxy=false` in `server.properties`.

From Mojang this feature was likely introduced to prevent any form of authentication proxies. So client modifications that allow to use Minecraft authentication provided by a third-party without actually receiving the credentials or proxying all traffic.

Reply:
I have attempted to rephrase it in https://github.com/games647/FastLogin/pull/811/commits/d7d0caf6e7499060277b677f46e6225c0ae7b9c4, but I am not sure whether it is enough. Please, check it out.